android/engine/src/test/java/org/smartregister/fhircore/engine/util/extension/DateTimeExtensionTest.kt

```diff
@@ -75,6 +75,14 @@ class DateTimeExtensionTest : RobolectricTest() {
 
     assertEquals("Feb", result)
   }
+  @Test
+  fun `Date asMmmm() should return correct formatted date`() {
+    val date = DateUtils.parseDate("2022-03-02", "yyyy-MM-dd")
+
+    val result = date.asMmmm()
+
+    assertEquals("March", result)
```

Comment:
Have you verified the returned date format is translatable?

Reply:
As in are the alphabetic words translated when you switch language for the application or update the device language?

Reply:
I think they should but kindly confirm.  Otherwise that can be addressed later shouldn't hinder us from merging this PR.